02.calendar/calendar.rb

```diff
@@ -2,46 +2,29 @@
 
 require 'date'
 require 'optparse'
-require 'Time'
 
-# 今日の日付
-date = Date.today
-# 本日の色付け
-# 今日の年
-y_date = date.year
-# 今日の月
-m_date = date.month
+todays = Date.today
+this_year = todays.year
+this_month = todays.month
 
-# オプション設定
 opt = OptionParser.new
-opt.on('-m [manth]', '月を指定します-m') { |m| m_date = m.to_i }
-opt.on('-y [year]', '年を指定します -y') { |y| y_date = y.to_i }
+opt.on('-m [manth]', '月を指定します-m') { |m| this_month = m.to_i }
```

Comment:
```suggestion
opt.on('-m [month]', '月を指定します -m') { |m| this_month = m.to_i }
```

typoがあるのと、スペースが入れた方が一貫性が出そうです